src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     class="search-form search-form_big"
+     data-qa= "big"
+    >
+      <label class="search-form__label search-form__label_big">
+        <input
+          class="search-form__input search-form__input-big"
+          name="input-big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
+      </label>
+    </form>
+
+    <form
+      class="search-form search-form_small"
+      data-qa="small"
+    >
+
+      <label class="search-form__label search-form__label_small">
+        <input
+        class="search-form__input search-form__input-small"
+        name="input-small"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
```suggestion
          class="search-form__input search-form__input-small"
          name="input-small"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
```
Consider fixing indents according to the code style guide